src/dmd/dtemplate.d

```diff
@@ -610,7 +611,9 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
                 s.parent = this;
 
                 /* Set isAliasSeq if this is of the form:
-                 * template AliasSeq(T...) { alias AliasSeq = T; }
+                 *   template AliasSeq(T...) { alias AliasSeq = T; }
+                 * or set isAlias if this is of the form:
```

Comment:
indentation

Reply:
is as intended. I prefer to set off code with a slight indentation in comments.